docs/spec/intro.mdx

```diff
@@ -1,23 +1,25 @@
 ---
 id: intro
-title: Overview
+title: Library
```

Comment:
Library doesnt provide any significant information to the end users. A title like Overview or Introduction, do. I suggest we keep the title as `LIbrary`